enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,325 @@
+
+* Backups should be taken without configuration after cluster installation from day 1.
+* Backups are saved to a default location on each master node's file system.
+* Backups are taken according to a default schedule.
+* Backups are being maintained according to a default retention policy.
+
+### Non-Goals
+
+* Save cluster backups to remote cloud storage (e.g. S3 Bucket).
+  - This could be a future enhancement or extension to the API.
+  - Having backups in an independent storage from OCP cluster is crucial for Disaster Recovery scenarios.
+* Automate cluster restoration.
+* Provide automated backups for non-self hosted architectures like Hypershift.
+
+## Proposal
+
+This work utilizes the [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) from current `CronJob` based implementation.
+Moreover, it uses the same [PeriodicBackupController](https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go#L81) as this proposal builds upon it, and also under the same feature gate.
+The default configuration is created through `config.openshift.io/v1alpha1 Backup` CR, with name `default`, to distinguish it from other user supplied `Backup` CRs.
+Using a CR enable users to override the default backup configuration.
+
+
+By design, the **No-Config** backup approach works alongside to the [Cron](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups) based method, and not a replacement, as shown in this [test](https://docs.google.com/document/d/1Ee9a7GJsae8EA7komPcEV72YlRgbSbR0B0KSo3XRBUc).
+Although both approaches use the same `config.openshift.io/v1alpha1 Backup` CR for configuration, the No-Config method does not utilise the `PVC` field in `Backup` CR. 
+In fact, configuring the PVC field for the No-Config method is being ignored as this [test](https://docs.google.com/document/d/1813mZo4KEfTWw_FRbmG8Un91oPA9-eF29FcrZk-RLxE) shows. 
+Instead, **No-Config** backups uses a default `hostPath` location on all control-plane nodes. 
+This ensures that backups are always accessible if any of the master node becomes unavailable.
+Using `hostPath` allows the **No-Config** to work on all variants of Openshift (e.g. Single-node and Bare-metal), as well as having backups on all control-plane nodes, for recovering from disaster scenarios.
+
+To sum up, the main two differences between the Cron-based and No-Config backups method are on storage and replication.
+The Cron-based stores the backups on the mounted PVC, while No-Config approach uses `hostPath` on all master nodes, regardless of any PVC configured on the `default` `config.openshift.io/v1alpha1 Backup` CR.
+
+### API Extensions
+
+This proposal uses the same [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) from the Cron based method.
+No additional API changes are required.
```

Comment:
I remember having conversation before about adding hostpath as an option to the existing API, so that this EP could leverage that config

Do you have an example of the CR as it would look for the default config?